packages.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "periapsisEngine",
+    "url": "https://github.com/Periapsis-Studios/Periapsis-Engine",
+    "method": "git",
+    "author": "Knedlik",
```

Comment:
You have to specifiy tags attributes, otherwise => "periapsisEngine has no tags"

Reply:
Should hopefully be okay now.